api/src/main/java/org/example/nominations/NominationRepository.java

```diff
@@ -3,5 +3,5 @@
 import org.example.nominations.models.Nomination;
 import org.springframework.data.repository.CrudRepository;
 
-public interface NominationRepository extends CrudRepository<Nomination,Long> {
+public interface NominationRepository extends CrudRepository<Nomination, Long> {
```

Comment:
Can we stop "Cleaning" the code base. it gets annoying to read through 50 files where the only change is one space.
We can do this right before the submission. so that we can just merge the change where it is just code formatting, rather than the formatting being mixed with changes that actually need to be reviewed

Reply:
👍